subcommittees/mentoring/roles/mentees.md

```diff
@@ -0,0 +1,17 @@
+# Mentees
+
+## Responsibilities
+
+
+## Time commitment
+
+
+## Requirements
+
+
+
+## To Volunteer
+
```

Comment:
Two notes:  

1) I don't think the "To Volunteer" language is appropriate here. Maybe "To Join" or "To Participate"  
2) For now, we'll just have people email me to join and I'll let them know that the program will be re-opening in March.  I'll update this once we have a better process in place.